wiki/Beatmap_Editor/Test_mode/id.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Test mode
+
+![Screenshot tampilan test mode](img/test-mode-example.jpg "Menggunakan test mode dengan autoplay")
+
+**Test mode** adalah sebuah fitur dari [beatmap editor](/wiki/Beatmap_Editor) yang mengizinkan pembuat beatmap untuk memainkan [beatmap](/wiki/Beatmap) sendiri mereka dari beatmap editor untuk dicoba. Mode ini bisi diakses melalui tombol `Test` di bawah-kanan editor.
```

Comment:
`mengizinkan pembuat beatmap untuk memainkan [beatmap](/wiki/Beatmap) sendiri mereka dari beatmap editor untuk dicoba.` 

-->

`memungkinkan para pembuat beatmap untuk dapat memainkan [beatmap](/wiki/Beatmap) milik mereka sendiri dari beatmap editor.`

Reply:
how about "pembuat beatmap agar dapat memainkan beatmap" instead? (untuk -> agar)

Reply:
both should be fine